### DISCUSSION TITLE
How to add Mozilla Firefox language to library json

### DISCUSSION THREAD

**Discussion by jcushenan** [EUCPilots/evergreen-module#575] [Q&A]
I have seen the notes here on Mozilla Firefox regarding language https://stealthpuppy.com/evergreen/issues/

How do I add this AppParam to a library json file? I am not sure what the syntax should be.

**Comment by aaronparker** [comment]
I'll have to update `Invoke-EvergreenLibraryUpdate` to support the `-AppParams` parameter
